ni_measurementlink_generator/ni_measurementlink_generator/template.py

```diff
@@ -146,6 +166,12 @@ def create_measurement(
 
     directory_out_path.mkdir(exist_ok=True, parents=True)
 
+    if annotations_description is None:
+        annotations_description = ""
+    if collection is None:
+        collection = ""
+    annotation_tags = json.dumps(tags)
```

Comment:
Why aren't we checking for None on the tags like the other two? Are there tests that we handle None for these three?

Reply:
json.dumps takes care of none... so I remove it.  
Yes there is one test.